spec/spec_helper_system.rb

```diff
@@ -19,7 +19,7 @@
 
     # Install modules and dependencies
     puppet_module_install(source: proj_root, module_name: 'splunk')
-    shell('puppet module install nanliu-staging --version 0.3.1')
+    shell('puppet module install puppet-archive --version 0.3.1')
```

Comment:
There's no such version of puppet/archive.

Reply:
Done.

Reply:
Hey @alexjfisher It does work but i might be accessing this the wrong way from the puppet archive module

![image](https://user-images.githubusercontent.com/6224270/30427662-39c67baa-9916-11e7-8c8a-7e02db478211.png)